### ISSUE TITLE
offline not working

### ISSUE THREAD

**Issue by tepecam18** [vite-pwa/nuxt#58]
Hi,

When I enable the "offline" option, I receive an error message saying "You are not connected to the internet." When I disable the "offline" option and stop the project, I get the error message (failed)net::ERR_CONNECTION_REFUSED. What should I do to make it work in an offline mode?

![image](https://github.com/vite-pwa/nuxt/assets/34689812/47904d79-e875-4819-8ac7-a8afb5f23511)

repo: https://github.com/tepecam18/my-app

**Comment by userquin** [comment]
@tepecam18 offline will not work in dev, you must run `build + preview` scripts in your local

**Comment by tepecam18** [comment]
@userquin  Thank you for your response. 

This was a repository I created because I couldn't share my main repository. I had deleted the configuration in the nitro routes section of my main project because it was giving an error. It seems that it is not working properly since nitro is not configured.

My issue has been resolved, so we can close the matter.

![image](https://github.com/vite-pwa/nuxt/assets/34689812/5f804e1b-73f5-4e2c-95ac-b39ec689d536)
